### DISCUSSION TITLE
Failing to remove DRM from recent purchases

### DISCUSSION THREAD

**Discussion by KaranthirMD** [noDRM/DeDRM_tools#243] [Q&A] [answered]
Hi, long-time user of DeDRM, first time I've encountered a problem I haven't been able to find a solution to. First off, thanks for maintaining this excellent product!

Today I synced two new Amazon purchases to my Kindle library in the PC app (I don't have a Kindle, so just use apps on PC and Android phone). However, I was unable to download the books. Not sure what the problem was, as the error message box that came up was too small to display it in full. I was able to download the books to my phone, so assumed the problem must have been that they couldn't be downloaded to PC version 1.17 due to some new interference from Amazon.
I downloaded the latest version of the Kindle app for PC and, to cover at least some bases, [disabled KFX support in Kindle for PC](https://www.epubor.com/how-to-deal-with-the-new-kindle-update.html). Now I can download the new books, and add them to Calibre (format: kfx-zip) but Calibre won't convert them (saying they are protected by DRM).
I am running the latest versions of Calibre and DeDRM. I converted a book with no problems earlier in the week (using Kindle 1.17), so the only thing I can think is that these new books are part of the First Reads promotion, although I've converted books from that with no problems in the past.

Any work arounds, thoughts, or anything I might have missed/be doing wrong? Thanks in advance.

**Comment by ElleKayEm** [comment, marked as answer]
This is a change on Amazon's end.  See notice at the top of this post: https://www.mobileread.com/forums/showthread.php?t=283371

**Reply by KaranthirMD** [reply]
Ah, not sure how I missed that. Thank you!

I don't have an e-ink device, and these are kfx files, so I will wait to see if a solution is found.

**Reply by ElleKayEm** [reply]
The last 2 times DeDRM was updated to deal with new KFX DRM, Amazon broke it within a day.  So you might want to look at getting a cheap refurbished Kindle.  Or return the books and buy elsewhere if possible.

**Reply by KaranthirMD** [reply]
They were Prime freebies, so not too big a deal. Just frustrating that I can't DeDRM them.

**Reply by jinmshu1** [reply]
I'm using Calibre v6.10 and DeDRM 10.0.3. I'm getting eBooks from a state library in the Kindle .awz3 format. I choose to download from Amazon using "Download & transfer via USB". I then add the eBook to Calibre and get the DRM error message before sending it to my Kindle Paperwhite. Been doing this for years and just recently have gotten the error message.

Any thoughts?

**Reply by ElleKayEm** [reply]
You chose the correct device when downloading?  You have that Kindle's serial number correctly entered in DeDRM's customization?

**Reply by jinmshu1** [reply]
OMG, thank you! You made my weekend!

Yes, I chose the right device, and I had added my new Kindle Paperwhite's serial number to DeDRM's plugin configuration.

Because of your response, I studied that serial number. In the last four digits, I had used O instead of 0. Correcting that solved my problem!

Thank you so much for the quick, concise reply. I'd donate to your website if you had one. Have a GREAT weekend and thanks again!

**Comment by nlaslett** [comment]
I'm seeing changes as well, even when using "Download & transfer via USB" with my old eInk paperwhite.  I haven't changed my setup, I'm running DeDRM 10.0.3, Calibre 6.14, and my paperwhite serial number is correctly entered.

Some books can be opened but not edited.  I get the calibre DRM warning on edit, but not on read.  I am able to convert (AZW3 -> AZW3) which basically makes a clean copy without the DRM.  Other books are completely fine on both read and edit.

Despite claims elsewhere, the Amazon website used to allow you to download borrowed books and de-DRM them.  They've removed that option from their interface.  Fair enough.

**Reply by ElleKayEm** [reply]
Were the copies you were trying to edit actually in your calibre library?  Because if you have DeDRM setup and you open a book which isn't in your library with the calibre ebook viewer, it will be decrypted for viewing but the DRM won't be removed until you add it to calibre.

**Reply by ElleKayEm** [reply]
DRM should be removed when you add the book to calibre if DeDRM was set up correctly at that time.  If a book still has DRM, then DeDRM will be triggered if you try to convert it.

**Reply by ElleKayEm** [reply]
I don't what claims you saw elsewhere about "download & transfer" for Kindle Unlimited/Prime Reading, but, yes, that formerly available option is gone on those books.  A good change for the authors.

**Reply by JSWolf** [reply]
> I don't what claims you saw elsewhere about "download & transfer" for Kindle Unlimited/Prime Reading, but, yes, that formerly available option is gone on those books. A good change for the authors.

We have our accounts linked and on the account that does not have KU, the KU eBooks did show up and D&T did show up but when I tried it, I was taken to a page that doesn't exist. So while you may see D&T as an option, it won't work.

**Reply by ElleKayEm** [reply]
You're talking about Family Library?  Can the account that doesn't have KU access those books otherwise?
